book/src/volume-health-monitor.md

```diff
@@ -0,0 +1,34 @@
+# Volume Health Monitory Feature
+
+## Status
+
+Status | Min K8s Version | Max K8s Version | CSI external-health-monitor-controller sidecar Version | CSI external-health-monitor-agent sidecar Version
+--|--|--|--|--
+Alpha | 1.19 | | 0.1.0 | 0.1.0
+
+## Overview
+
+The External Health Monitor is part of Kubernetes implementation of [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec). It was introduced as an Alpha feature in Kubernetes v1.19.
+
+The [External Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) is impelmeted as two components: `External Health Monitor Controller` and `External Health Monitor Agent`.
+
+- External Health Monitor Controller:
```

Comment:
Can you describe what these two components do after they make the csi calls?

Reply:
Done